src/entities/Votes/model.ts

```diff
@@ -27,18 +29,26 @@ export default class VotesModel extends Model<VoteAttributes> {
   }
 
   static async createEmpty(proposal_id: string) {
+    const newScore = this.newScoreFor(proposal_id)
+    await this.create(newScore)
+    return newScore
+  }
+
+  static async safelyGetVotesFor(proposal: ProposalAttributes<any>) {
```

Comment:
some notes:
- we don't use the subfix `for` 
- it is prefer use `proposal_id: string` instead of the whole proposal
- this class is a direct connection with the database, isn't its responsibility know what is safety and what not

Reply:
Moved to Votes/routes